libchannelserver/ChannelRPCServer.cpp

```diff
@@ -584,20 +584,11 @@ void dev::ChannelRPCServer::onClientTopicRequest(
         Json::Value topicsValue = root;
         if (!topicsValue.empty())
         {
-            bool versionGt2 = (g_BCOSConfig.version() > dev::VERSION::V2_0_0);
             for (size_t i = 0; i < topicsValue.size(); ++i)
             {
                 std::string topic = topicsValue[(int)i].asString();
                 CHANNEL_LOG(TRACE) << "onClientTopicRequest insert" << LOG_KV("topic", topic);
-                if (topic.substr(0, topicNeedCertPrefix.size()) == topicNeedCertPrefix &&
-                    versionGt2)
-                {
-                    topics->insert(std::make_pair(topic, TopicStatus::VERIFYING_STATUS));
-                }
-                else
-                {
-                    topics->insert(std::make_pair(topic, TopicStatus::VERIFYI_SUCCESS_STATUS));
-                }
+                topics->insert(std::make_pair(topic, TopicStatus::VERIFYI_SUCCESS_STATUS));
```

Comment:
I think 50 lines is enough to modify topics' type to vector<string>, maybe you can try it?

Reply:
I have changed it to set<string> for session and service topics.
but for lines limit.I will push it in next pr.

Reply:
> I have changed it to set for session and service topics.
> but for lines limit.I will push it in next pr.

> I have changed it to set for session and service topics.
> but for lines limit.I will push it in next pr.

Do you mean that the changes you made more than 50 lines?